spec/integration/submitting_content_item_spec.rb

```diff
@@ -183,7 +183,7 @@
 
       content_item = ContentItem.find_by(base_path: "/vat-rates")
       expect(content_item.publishing_scheduled_at).to be_within(0.001.seconds).of(publish_time)
-      expect(content_item.scheduled_publishing_delay_seconds).to eq(25)
+      expect(content_item.scheduled_publishing_delay_seconds).to be_between(0, 25).inclusive
```

Comment:
It would be better to do `be_between(24, 25).inclusive` because if it's 0 then something is broken.